### ISSUE TITLE
kubernetes-helm fails to build

### ISSUE THREAD

**Issue by fxcoudert** [Homebrew/homebrew-core#31486]
`kubernetes-helm` fails to build on macOS 10.14 Mojave (https://jenkins.brew.sh/job/Homebrew%20Mojave%20Testing/488/version=mojave_testing/testReport/junit/brew-test-bot/mojave/install_kubernetes_helm/) #28817 with the following error:

```
�[0;32m[INFO]	�[mReplacing existing vendor dependencies
�[0;32m[INFO]	�[mRemoving nested vendor and Godeps/_workspace directories...
�[0;32m[INFO]	�[mRemoving: /private/tmp/kubernetes-helm-20180826-10408-owbz12/src/k8s.io/helm/vendor/github.com/cyphar/filepath-securejoin/vendor
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x30 pc=0x12fca82]

goroutine 1 [running]:
github.com/Masterminds/glide/path.StripVendor.func1(0xc420592e10, 0x81, 0x0, 0x0, 0x14f11a0, 0xc420181f50, 0x0, 0x0)
	/private/tmp/glide-20180816-23735-1ovdnmf/glide-0.13.1/src/github.com/Masterminds/glide/path/strip.go:28 +0x42
path/filepath.walk(0xc4205cfe80, 0x76, 0x14f8320, 0xc4205e7930, 0xc420442360, 0x0, 0x0)
	/usr/local/Cellar/go/1.10.3/libexec/src/path/filepath/path.go:377 +0x20d
path/filepath.walk(0xc4205d2770, 0x6f, 0x14f8320, 0xc4205e7380, 0xc420442360, 0x0, 0x0)
	/usr/local/Cellar/go/1.10.3/libexec/src/path/filepath/path.go:381 +0x2c2
path/filepath.walk(0xc420031ec0, 0x5b, 0x14f8320, 0xc4205e72b0, 0xc420442360, 0x0, 0x0)
	/usr/local/Cellar/go/1.10.3/libexec/src/path/filepath/path.go:381 +0x2c2
path/filepath.walk(0xc420031440, 0x54, 0x14f8320, 0xc4202c12b0, 0xc420442360, 0x0, 0x0)
	/usr/local/Cellar/go/1.10.3/libexec/src/path/filepath/path.go:381 +0x2c2
path/filepath.walk(0xc420387180, 0x49, 0x14f8320, 0xc42009e0d0, 0xc420442360, 0x0, 0x20)
	/usr/local/Cellar/go/1.10.3/libexec/src/path/filepath/path.go:381 +0x2c2
path/filepath.Walk(0xc420387180, 0x49, 0xc420442360, 0xc42029dee0, 0x0)
	/usr/local/Cellar/go/1.10.3/libexec/src/path/filepath/path.go:403 +0x106
github.com/Masterminds/glide/path.StripVendor(0x14ae577, 0x3b)
	/private/tmp/glide-20180816-23735-1ovdnmf/glide-0.13.1/src/github.com/Masterminds/glide/path/strip.go:22 +0x10a
github.com/Masterminds/glide/action.Install(0xc4201bb540, 0x149a201)
	/private/tmp/glide-20180816-23735-1ovdnmf/glide-0.13.1/src/github.com/Masterminds/glide/action/install.go:62 +0x35b
main.commands.func12(0xc420214420, 0x0, 0xc420214420)
	/private/tmp/glide-20180816-23735-1ovdnmf/glide-0.13.1/src/github.com/Masterminds/glide/glide.go:510 +0x2bf
github.com/Masterminds/glide/vendor/github.com/codegangsta/cli.HandleAction(0x140f100, 0x14b8b40, 0xc420214420, 0xc4201cc300, 0x0)
	/private/tmp/glide-20180816-23735-1ovdnmf/glide-0.13.1/src/github.com/Masterminds/glide/vendor/github.com/codegangsta/cli/app.go:490 +0xc8
github.com/Masterminds/glide/vendor/github.com/codegangsta/cli.Command.Run(0x149859e, 0x7, 0x1496c20, 0x1, 0x0, 0x0, 0x0, 0x14a3b3b, 0x20, 0x0, ...)
	/private/tmp/glide-20180816-23735-1ovdnmf/glide-0.13.1/src/github.com/Masterminds/glide/vendor/github.com/codegangsta/cli/command.go:210 +0xa36
github.com/Masterminds/glide/vendor/github.com/codegangsta/cli.(*App).Run(0xc420202000, 0xc42001e0c0, 0x3, 0x3, 0x0, 0x0)
	/private/tmp/glide-20180816-23735-1ovdnmf/glide-0.13.1/src/github.com/Masterminds/glide/vendor/github.com/codegangsta/cli/app.go:255 +0x6a0
main.main()
	/private/tmp/glide-20180816-23735-1ovdnmf/glide-0.13.1/src/github.com/Masterminds/glide/glide.go:104 +0x5a9
make: *** [bootstrap] Error 2
```

**Comment by SConaway** [comment]
Log: [https://gist.github.com/73afecc73bbbd3cdea87c4cdf2209207](url)

**Comment by fxcoudert** [comment]
With newer glide bottles, it now fails to build on all macOS CI machines: https://github.com/helm/helm/issues/4584

**Comment by crivera** [comment]
Getting the same error any update?

**Comment by mattfarina** [comment]
I think the error is happening because the following patch in the glide.rb is not applied...

https://github.com/Homebrew/homebrew-core/blob/a2aa4aecd954223776009a42316fb95d09318716/Formula/glide.rb#L21-L24

I'm hoping the PR at https://github.com/Homebrew/homebrew-core/pull/32474 will fix it (glide 0.13.2) by not needing to have a patch.
